{
  "_annotation": {
    "cve_id": "CVE-2025-27401",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2025/CVE-2025-27401.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:enalean:tuleap:*:*:*:*:community:*:*:*",
                "matchCriteriaId": "5609916B-3E6B-4009-8837-701EF159FF6F",
                "versionEndExcluding": "16.4.99.1740498975",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      },
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:enalean:tuleap:*:*:*:*:enterprise:*:*:*",
                "matchCriteriaId": "54A72640-277F-4E54-A2F4-8948FF0F16E2",
                "versionEndExcluding": "16.4-6",
                "versionStartIncluding": "16.4",
                "vulnerable": true
              },
              {
                "criteria": "cpe:2.3:a:enalean:tuleap:*:*:*:*:enterprise:*:*:*",
                "matchCriteriaId": "44FFF09C-980B-4D7E-AF34-BBAECB1B179D",
                "versionEndExcluding": "16.3-11",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}